### PR TITLE
Feature/show routes based off checkboxes

### PR DESCRIPTION
**Pull request by halee010**
Front end piece which includes: 
- Resizing map based on resize of browser window
- Sidebar which includes checkboxes
- Checkboxes which filter out the routes displayed

It needs a lot of CSS TLC but the structure is here.

rev: @EddyIonescu

### CODE REVIEW

File: src/styles/App.css [hunk 1]
```diff
@@ -1,3 +1,9 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins', sans-serif;
```
Comment (EddyIonescu) [inline src/styles/App.css]: +1

File: src/styles/App.css [hunk 2]
```diff
@@ -31,4 +37,42 @@
...
   background-color: #ffffffa6;
   border: 1px solid gray;
-}
\ No newline at end of file
+}
+
+.map {
+  padding: 0;
+}
+
```
Comment (EddyIonescu) [inline src/styles/App.css]: really nice CSS styles

File: src/Map.jsx [hunk 3]
```diff
@@ -38,17 +35,87 @@ class Map extends Component {
         coordinates: { lon: 0, lat: 0 },
         info: { vid: '', heading: 0 },
       },
+      geojson: muniRoutesGeoJson,
     };
   }
 
+  componentWillMount() {
```
Comment (EddyIonescu) [inline src/Map.jsx]: If possible, let's use componentDidMount as then it'll only run once - and it's better practice to initialize things (lines 43 and 45) in the constructor. See https://github.com/facebook/react/issues/7671

File: src/Map.jsx [hunk 4]
```diff
@@ -38,17 +35,87 @@ class Map extends Component {
...
+    this.setState({
+      viewport: {
+        width: window.innerWidth,
+        height: window.innerHeight,
+        longitude: -122.41669,
+        latitude: 37.7853,
+        zoom: 15,
+        pitch: 0,
+        bearing: 0,
+      },
```
Comment (EddyIonescu) [inline src/Map.jsx]: this makes the zoom in/out disappear - make sure this works before merging

File: package.json [hunk 5]
```diff
@@ -11,6 +11,7 @@
     "babel-loader": "7.1.1",
     "babel-preset-react-app": "^3.0.3",
     "babel-runtime": "6.26.0",
+    "bootstrap": "4.0.0-beta",
```
Comment (EddyIonescu) [inline package.json]: Could you describe in the PR how adding bootstrap/reactstrap is useful? (is it to make the sidebar prettier?)

File: src/Map.jsx (hunk 4 above)
Comment (halee010) [reply src/Map.jsx]: Ahh! Sorry for the lag. Will fix this ASAP!

**Pull request merged**